### PR TITLE
docs(cli): Update CLI output text for clarity

### LINKED ISSUE #921: fix: Updating the CLI template to guide the user towards opening the project after creating
**Description**
Currently after creating an application, it doesn't guide users to go into the respective directly and run the code.

**Additional Context**

BEFORE
```
Created a Very Good App! 🦄

+----------------------------------------------------+
| Looking for more features?                         |
| We have an enterprise-grade solution for companies |
| called Very Good Start.                            |
|                                                    |
| For more info visit:                               |
| https://verygood.ventures/solution/very-good-start |
+----------------------------------------------------+
```
AFTER
```
Created a Very Good App! 🦄

In order to open your project, type:
       $ cd app_name
       $ code .
+----------------------------------------------------+
| Looking for more features?                         |
| We have an enterprise-grade solution for companies |
| called Very Good Start.                            |
|                                                    |
| For more info visit:                               |
| https://verygood.ventures/solution/very-good-start |
+----------------------------------------------------+
```

### PR DESCRIPTION
**Pull request by yaksh1**
Resolves #921

Updated CLI template output for 
``` very_good create flutter_app my_app```
for more clarity on hoe to open the project.

```
Created a Very Good App! 🦄

In order to open your project, type:
       $ cd app_name
       $ code .
+----------------------------------------------------+
| Looking for more features?                         |
| We have an enterprise-grade solution for companies |
| called Very Good Start.                            |
|                                                    |
| For more info visit:                               |
| https://verygood.ventures/solution/very-good-start |
+----------------------------------------------------+
```

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [x] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [x] 🗑️ Chore

### CODE REVIEW

File: lib/src/commands/create/templates/very_good_core/very_good_core_template.dart [hunk 1]
```diff
@@ -19,14 +19,19 @@ class VeryGoodCoreTemplate extends Template {
...
+  void _logSummary(Logger logger, Directory outputDir) {
     logger
       ..info('\n')
       ..created('Created a Very Good App! 🦄')
       ..info('\n')
+      ..info('In order to open your project, type:')
+      ..info('\n')
+      ..info('  \$ cd ${outputDir.path}')
+      ..info('\n')
+      ..info(r'  $ code .')
```
Comment (alestiago) [inline lib/src/commands/create/templates/very_good_core/very_good_core_template.dart:34]: I think this `$ code .` assumes the user is using Visual Studio Code. I would personally suggest users to refer to the README.md for "getting started" instructions.

---

Note that Flutter create does not refer to `code .`:

```txt
Creating project hello_world...
Resolving dependencies in hello_world... (1.3s)
Got dependencies in hello_world.
Wrote 129 files.

All done!
You can find general documentation for Flutter at: https://docs.flutter.dev/
Detailed API documentation is available at: https://api.flutter.dev/
If you prefer video documentation, consider: https://www.youtube.com/c/flutterdev

In order to run your application, type:

  $ cd hello_world
  $ flutter run

Your application code is in hello_world/lib/main.dart.
```

Comment (tomarra) [reply lib/src/commands/create/templates/very_good_core/very_good_core_template.dart:34]: Yeah I agree with @alestiago here. This assumes an IDE and that they have `code` setup in their path.

Comment (yaksh1) [reply lib/src/commands/create/templates/very_good_core/very_good_core_template.dart:34]: @alestiago @tomarra So should I suggest users to refer ```Quick Start``` or ```commands``` section in the README.md?
The problem with ```flutter run``` is that when an  app is created using very_good_cli it doesn't generate a main.dart file so its unable to run it.

We could do something like:

Created a Very Good App! 🦄

for more commands to get started, refer:
       <i>the section in readme</i>

Comment (alestiago) [reply lib/src/commands/create/templates/very_good_core/very_good_core_template.dart:34]: What about:

```txt
Created a Very Good App! 🦄

To get started refer to app_name/README.md.
      
Your application is located in desktop/app_name.
```

Comment (yaksh1) [reply lib/src/commands/create/templates/very_good_core/very_good_core_template.dart:34]: Yeah that sounds more clear and straight forward on where my project is as well as what is the next step. Ill update the code accordingly. Thank you for the guidance.

Comment (yaksh1) [reply lib/src/commands/create/templates/very_good_core/very_good_core_template.dart:34]: @alestiago can you please elaborate difference between Directory.current and Directory.path

**Review by alestiago — APPROVED** [review]
LGTM.

I made some amendments to ensure the paths are not using a platform specific separator and included some tests.

This is the output I'm seeing:

</img>

**Review by wolfenrain — APPROVED** [review]

**Comment by yaksh1** [comment]
@alestiago Thank you for these amendments, It was fun working on this. Also, I have some ideas for very_good_cli, if we could discuss it, I would really appreciate it.

**Comment by alestiago** [comment]
Hi @yaksh1 ! I'm glad you enjoyed working on this. If you have some enhancement ideas for the tool feel free to open an issue for each one of them and we can discuss it there 🙌

**Pull request merged**